usersound.properties.example

```diff
@@ -0,0 +1,4 @@
+//Here we can map a user to a sound file, so many users can share the same intro soundbite and/or so you don't have to have extra copies of the same sound file
+
+//Discord user and the file (with extension)
+Queen=WeAreTheChampions.mp3
```

Comment:
Does this work with nested filepaths? If the folder is specified as /path/to/folder and the file is in /path/to/folder/more/song.mp3, would the value in this file be more/song.mp3?